coffee_lib/src/plugin.rs

```diff
@@ -125,19 +126,24 @@ impl Plugin {
     /// configure the plugin in order to work with cln.
     ///
     /// In case of success return the path of the executable.
-    pub async fn configure(&mut self, verbose: bool) -> Result<String, CoffeeError> {
+    pub async fn configure(
+        &mut self,
+        root_path: &str,
+        verbose: bool,
+    ) -> Result<String, CoffeeError> {
         let exec_path = if let Some(conf) = &self.conf {
             if let Some(script) = &conf.plugin.install {
-                sh!(self.root_path.clone(), script, verbose);
+                eprintln!("installing plugin...");
```

Comment:
```suggestion
```

we have the logger for this, but we do not need it

Reply:
yes that slipped
I didn't mean to push this